src/desktop/components/frame_animator/test/index.jest.js

```diff
@@ -1,45 +1,46 @@
-import sinon from "sinon"
+import FrameAnimator from "../index"
 
-const rewire = require("rewire")("../index")
-const FrameAnimator = rewire.default
```

Comment:
This (unrelated) spec started breaking on this test run as well, with a `rewire` related error

Converting it to Jest and getting rid of the rewire stuff was sufficient to get this green again.